Frontify/guideline-blocks
# feat(shared): provide default tokens in `useGuidelineDesignTokens`

Provide default values for all guideline tokens. Default values are collected from the UI when creating a new guideline.

How to test:
1. Serve a block, which uses the `useGuidelineDesignTokens` hook, locally (for examle the text block)
2. Check if the styles of headings, buttons etc. are matching the design tokens in the sidebar.
3. Check it also in a brand new guideline. You can create a new guideline on product.frontify.com.

Related ticket: https://app.clickup.com/t/8677fr17t
